app/components/post_textbox/post_textbox.js

```diff
@@ -175,8 +185,22 @@ export default class PostTextbox extends PureComponent {
         }
     };
 
-    handleAndroidKeyboard = () => {
-        this.blur();
```

Comment:
when using the device keyboard instead of a hardware keyboard and then I use the hardware button to hide the keyboard the textbox won't get blurred.

Reply:
Hello @enahum,

I ran into an issue when I enabled the blurring when soft-keyboard hides.

Here's the scenario (this only happens in Android):
1. Connect to a bluetooth keyboard.
2. Enable virtual-keyboard to be visible (by tapping keyboard icon bottom-right corner and enable Virtual Keyboard).
3. Tap the TextInput box to focus.
4. Virtual-keyboard will be shown.
5. Type something with hardware keyboard.
6. Virtual-keyboard will auto-hide and TextInput will be blurred.
7. User cannot type anything using the hardware keyboard as this will continue to happen.

The workaround scenario I tried to do was to add a keyboardState state of 'hiding' and check it in onChangeText(as this gets called when input is received before virtual-keyboard is hidden) and handle it in keyboardWillHide. But keyboardWillHide isn't supported on Android platform.

At this point, all I can think of as a solution is to receive every input from Android native side and enable focus when any key is pressed but this means native side will send message every time a key is pressed. This eventually lead to performance issue as it will need to parse every key press.

I'll need some help to resolve this issue.

Thank you in advance!

Reply:
Update: I'm not able to reproduce the issue I described above.
I will test more though.